### PR TITLE
Chorng/update contribution markdown

### PR DESCRIPTION
**Pull request by chorng**

### CODE REVIEW

File: collections/cams_glc_2017.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 Name: CAMS_GLC_2017
-OpenEOPID: CAMS_GLC
```
Comment (dthiex) [inline collections/cams_glc_2017.yaml:2]: As you put all this changes in a separate PR (which is very good) this PR should only contain the updates to the `CONTRIBUTING.md`.

**Review by dthiex — CHANGES_REQUESTED** [review]

File: collections/cams_glc_2017.yaml (hunk 1 above)
Comment (chorng) [reply collections/cams_glc_2017.yaml]: Fixed. I forgot to switch back to main first.

**Comment by chorng** [comment]
Could you please help edit this markdown? We don't know about the structure back there and we are not included in the loop for update, e.g., the time interval update you mentioned in the other PR. It is really difficult for me to compile the contributing markdown correctly with the latest update. It is also difficult to create a ymal for new data collections.

**Comment by Dorothyrono** [comment]
@dthiex  could you please do a final review, i think we made the changes from our side already

**Review by dthiex — APPROVED** [review]
